cmd/sonrd/main.go

```diff
@@ -43,8 +40,7 @@ func main() {
 		app.New,
 		// this line is used by starport scaffolding # root/arguments
 	)
-	// rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.cli.yaml)")
```

Comment:
Why removed

Reply:
Wasnt me was like this before - https://github.com/sonr-io/sonr/blob/04960e15fed064a7f6ebdf2c7ab82242dc6cf0fa/cmd/sonrd/main.go#L46